src/client/Microsoft.Identity.Client/Platforms/Features/WebView2WebUi/WebView2WebUiFactory.cs

```diff
@@ -1,23 +1,19 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using Microsoft.Identity.Client.ApiConfig.Parameters;
 using Microsoft.Identity.Client.Internal;
 using Microsoft.Identity.Client.Platforms.Features.DesktopOs;
-using Microsoft.Identity.Client.Platforms.Features.WebView2WebUi;
+using Microsoft.Identity.Client.Platforms.Features.WinFormsLegacyWebUi;
 using Microsoft.Identity.Client.Platforms.Shared.Desktop.OsBrowser;
 using Microsoft.Identity.Client.UI;
 using Microsoft.Web.WebView2.Core;
 
-namespace Microsoft.Identity.Client.Platforms.net5win
+namespace Microsoft.Identity.Client.Platforms.Features.WebView2WebUi
 {
-    internal class Net5WebUiFactory : IWebUIFactory
+    internal class WebView2WebUiFactory : IWebUIFactory
```

Comment:
Combined `MsalDesktopWebUiFactory` and `Net5WebUiFactory` into `WebView2WebUiFactory` because the code was exactly the same (except the Net5 attribute). Put it in `Features.WebView2WebUi` so it can be correctly only referenced in Net5 and Msal.Desktop.